src/theme/NavbarItem.tsx

```diff
@@ -1,24 +1,49 @@
 import React from 'react'
 import OriginalNavBarItem from '@theme-original/NavbarItem'
 import { useLocation } from '@docusaurus/router'
+import { TraceEvent } from '@uniswap/analytics'
+import { BrowserEvent, EventName } from '@uniswap/analytics-events'
 
-export default function NavbarItem(props: { className: string }) {
+enum ProtocolVersion {
+  V1 = 'V1',
+  V2 = 'V2',
+  V3 = 'V3',
+}
+
+const getElementName = (version: string, elementName: string) => {
+  if (elementName in ProtocolVersion) {
+    return elementName
+  }
+  return version + ' ' + elementName
```

Comment:
Per offline discussion: we should break this out into a separate field for better/easier analytics dashboards